src/components/SearchBar.test.ts

```diff
@@ -3,36 +3,36 @@ import { mount } from '@vue/test-utils';
 import SearchBar from './SearchBar.vue';
 
 describe('Searchbar component', () => {
-  test('it has a label', () => {
-    const wrapper = mount(SearchBar);
-    expect(wrapper.find('label').exists()).toBe(true);
-  });
+  // test('it has a label', () => {
```

Comment:
Could we just remove this test, rather than commenting it?

Reply:
Yes, of course!